.changeset/healthy-coins-fold.md

```diff
@@ -0,0 +1,8 @@
+---
+"@namehash/namekit-react": minor
+"@namehash/nameguard-react": patch
+---
+
+New Style Addition: Updated components/Link.tsx to include a new size class, xsmall (nk-text-xs), to enhance design compatibility and provide more styling options.
+Folder Structure Update: Introduced two new folders, /client and /. The /client folder exports components that require "use client" at the top of their files, while the / folder exports components that do not require this directive.
```

Comment:
@notrab Hey can you please review and sign off on this? It seems surprising to me that we have to do this. Please share your advice so that we can merge this PR.